Packs/ShiftLeft/Integrations/shiftleft/shiftleft.yml

```diff
@@ -0,0 +1,84 @@
+category: Vulnerability Management
+commonfields:
+  id: ShiftLeft CORE
+  version: -1
+configuration:
+- display: ShiftLeft Organization Id
+  name: org_id
+  type: 0
+  required: true
+  additionalinfo: You can find this under Account Settings -> Org ID
+- display: ShiftLeft Access Token
+  name: access_token
+  type: 4
+  required: true
+  additionalinfo: You can find this under Account Settings -> Access Token
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Integrate ShiftLeft CORE code analysis platform with Cortex XSOAR.
+display: ShiftLeft CORE
+name: ShiftLeft CORE
+script:
+  script: "-"
+  type: python
+  commands:
+  - name: shiftleft-list-apps
+    outputs:
+    - contextPath: ShiftLeft
+      description: API response
+    description: List Apps
```

Comment:
Your outputs are not correct. It seems (from the code) that you return an ID, so you should define it as an output. If you have more outputs you should add them all. Please see here: https://xsoar.pan.dev/docs/integrations/context-and-outputs#integration-yml